obspy/io/nordic/core.py

```diff
@@ -349,21 +344,31 @@ def _read_origin(line):
         except ValueError:
             new_event.origins[0].__dict__[key] = None
     if new_event.origins[0].depth:
-        new_event.origins[0].depth *= 1000
+        new_event.origins[0].depth *= 1000.
     if line[43].strip():
         warnings.warn("Depth indicator {0} has not been mapped "
                       "to the event".format(line[43]))
     if line[44].strip():
         warnings.warn("Origin location indicator {0} has not been mapped "
                       "to the event".format(line[44]))
+    if line[10] == "F":
+        new_event.origins[0].time_fixed = True
     new_event.creation_info = CreationInfo(agency_id=line[45:48].strip())
     new_event.origins[0].creation_info = CreationInfo(
         agency_id=line[45:48].strip())
-    ksta = Comment(text='Number of stations=' + line[49:51].strip())
-    new_event.origins[0].comments.append(ksta)
-    if _float_conv(line[51:55]) is not None:
+    used_station_count = line[49:51].strip()
+    if used_station_count != '':
+        new_event.origins[0].comments.append(
```

Comment:
What is the purpose of this comment if the information is also in the `OriginQuality`?

Reply:
Good question: it was previously put in as a comment (before this patch) and I left it there just in case anyone was expected it to be there - I would rather remove it though.

Reply:
I don't use seisan anymore but I would be in favor of removing it. Relying on a comment to extract information is a pretty dicey practice anyhow.

Reply:
Agreed - I think I just didn't see where it should go when I first wrote this.  Keen to remove, I might just put a note in the doc-string as a change for 1.2.0.